tests/ares_test.py

```diff
@@ -5,91 +5,85 @@
 
 import io
 import os
+import pathlib
 from unittest import TestCase
 
+import requests
+from requests.status_codes import codes
+
 import responses
 
-from ares_util import ares
 from ares_util.ares import call_ares
 from ares_util.exceptions import AresConnectionError, AresServerError
 from ares_util.helpers import normalize_company_id_length
 from ares_util.settings import ARES_API_URL
 
 
-def _read_mock_response(filename):
-    _this_dir = os.path.dirname(os.path.realpath(__file__))
-
-    return io.open(os.path.join(_this_dir, "responses", filename), mode='r', encoding='utf-8').read()
+def _json_response(company_id):
+    this_dir = pathlib.Path(os.path.realpath(__file__)).parent
+    return (this_dir / "responses" / f"{company_id}.json").read_text()
 
 
 class CallARESTestCase(TestCase):
     def test_invalid_values(self):
-        invalid_values = [False, True, 42, -42, "foo"]
+        invalid_values = [False, True, 42, -42, "foo", 25596640]
 
         for invalid_value in invalid_values:
             self.assertFalse(call_ares(invalid_value))
 
     @responses.activate
-    def test_numerically_valid(self):
+    def test_not_found(self):
         company_id = 25596641
-        responses.add(responses.GET, '{0}?ico={1}'.format(ARES_API_URL, company_id), match_querystring=True, body=_read_mock_response('{}.xml'.format(company_id)))
+        responses.get(f'{ARES_API_URL}{company_id}', body=_json_response(company_id), status=codes.not_found)
 
         self.assertFalse(call_ares(company_id=company_id))
         self.assertEqual(1, len(responses.calls))
 
     @responses.activate
     def test_encoding(self):
-        company_id = 68407700
-        responses.add(responses.GET, '{0}?ico={1}'.format(ARES_API_URL, company_id), match_querystring=True, body=_read_mock_response('{}.xml'.format(company_id)))
+        company_id = '68407700'
+        responses.get(f'{ARES_API_URL}{company_id}', body=_json_response(company_id))
 
         ares_response = call_ares(company_id=company_id)
 
         self.assertEqual(ares_response['legal']['company_name'], "České vysoké učení technické v Praze")
-        self.assertEqual(ares_response['address']['street'], "Zikova 1903/4")
-        self.assertEqual(1, len(responses.calls))
-
-    @responses.activate
-    def test_special_case_for_issue9(self):
```

Comment:
I guess Issue #9 was about a difference between `PSC` and `PSČ` and is not relevant anymore?

Reply:
That's right.